docs/source/endpoints/translations.md

```diff
@@ -92,7 +92,6 @@ To unlink the content, issue a `DELETE` request on the `@translations` endpoint
 :language: http
 ```
 
-
```

Comment:
Please do not remove the extra blank lines because they help authors identify headings in a wall of text.